pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -977,6 +1044,31 @@ func getPVCOSDVolumes(osdProps *osdProperties) []v1.Volume {
 		volumes = append(volumes, metadataPVCVolume...)
 	}
 
+	// If we have a waldata PVC let's add it
+	if osdProps.onPVCWithWaldata() {
+		waldataPVCVolume := []v1.Volume{
+			{
+				Name: osdProps.waldataPVC.ClaimName,
+				VolumeSource: v1.VolumeSource{
+					PersistentVolumeClaim: &osdProps.waldataPVC,
+				},
+			},
+			{
+				// We need a bridge mount which is basically a common volume mount between the non privileged init container
+				// and the privileged provision container or osd daemon container
+				// The reason for this is mentioned in the comment for getPVCInitContainer() method
+				Name: fmt.Sprintf("%s-bridge", osdProps.waldataPVC.ClaimName),
```

Comment:
For raw mode I thought we didn't need the bridge either

Reply:
the same comments with above.

Reply:
set it resolved after rebased, waiting for new comments.